components/navbar/index.tsx

```diff
@@ -0,0 +1,31 @@
+import Link from "next/link";
+import styles from "./navbar.module.scss";
+
+const Navbar = () => {
+  return (
+    <div className={styles.container}>
+      <Link href="/" className={styles.title}>
+        Lorem
+      </Link>
+      <ul>
+        <li>
+          <Link href="/">Home</Link>
+        </li>
+        <li>
+          <Link href="/package">Package</Link>
+        </li>
+        <li>
+          <Link href="/series">Series</Link>
+        </li>
+        <li>
+          <Link href="/signin">Sign in</Link>
+        </li>
+        <li>
+          <Link href="/signout">Sign out</Link>
+        </li>
+      </ul>
+    </div>
+  );
+};
+
```

Comment:
Link tag is not required. Next js works differently. The pages folder is responsible for routing. This should only contain the <li> tags, now futher this component should be used in the pages/{respective route files} to render the data.